### DISCUSSION TITLE
请问一下paddlepaddle==3.0.0b2和paddle-custom-npu==3.0.0b2应该使用那个版本的paddleocr以及paddleclas

### DISCUSSION THREAD

**Discussion by Stefan3Zz** [PaddlePaddle/PaddleOCR#14333] [Q&A]
请问一下我在昇腾NPU 910b上想使用paddleocr，我的paddlepaddle==3.0.0b2 -i https://www.paddlepaddle.org.cn/packages/stable/cpu/，以及paddle-custom-npu==3.0.0b2 -i https://www.paddlepaddle.org.cn/packages/stable/npu/ 是上述版本的，我需要安装paddleocr和paddleclas，应该安装什么版本的，我目前安装的是"paddleocr=2.9.1"，paddleclas=2.6.0，在这个版本下一但加载paddleOcr，把use_npu=True这个参数加上，就会在加载的时候报错，我的报错记录在图片里，以及我加载模型的方式，希望大佬们解答一下
![20241205-160820](https://github.com/user-attachments/assets/7ba47dc4-266e-47df-aeb2-4f829ece1255)
![20241205-160824](https://github.com/user-attachments/assets/ed2da7f1-c549-4364-8590-821f9e211580)

**Comment by GreatV** [comment]
这个版本应该是可以的，不行的话试试3.0.0b1

**Reply by Stefan3Zz** [reply]
当我执行如下的命令的时候是正常的，是否可以排除是驱动的问题
![Uploading 20241205-161540.jpeg…]()

**Reply by GreatV** [reply]
不用npu可以跑吗

**Reply by Stefan3Zz** [reply]
不用npu是只把use_npu=True这个参数去除掉吗，是可以运行的，但是我不确定他是否运行在npu上

**Reply by Stefan3Zz** [reply]
![20241205-161540](https://github.com/user-attachments/assets/21fa1871-7c44-43af-bc89-e37f8b5bb1b3)

**Reply by Stefan3Zz** [reply]
因为目前在cpu上我用paddleocr来处理一张图片，需要30s左右的耗时，所以我想看看在npu上是否能更快一点

**Reply by GreatV** [reply]
这个问题建议提交到 PaddlePaddle/PaddleCustomDevice

**Reply by Stefan3Zz** [reply]
请问我应该怎么移动这个问题到你那个位置

**Reply by GreatV** [reply]
需要use_npu=True，可以试试不同padde框架版本

**Reply by GreatV** [reply]
在那个仓库提个issue吧

**Reply by Stefan3Zz** [reply]
已经在那个仓库提问啦，可以帮忙找人来解答一下吗

**Reply by Stefan3Zz** [reply]
> [GreatV ](/GreatV) [ Dec 5, 2024 ](#discussioncomment-11470259) 
> Maintainer

请问有比较成熟，并且有pythondemo的padde框架推荐吗，可以在910b的npu上运行的

**Comment by GreatV** [comment]
https://paddlepaddle.github.io/PaddleOCR/latest/model/hardware/install_other_devices.html

**Reply by Stefan3Zz** [reply]
有arm64的系统架构的镜像吗
